src/state/languages.js

```diff
@@ -56,11 +58,14 @@ export const reducer = (state = initialState, action) => {
   }
 };
 
-export const load = () => dispatch => {
-  dispatch({ type: LOADING });
+export const load = () => (dispatch, getState) => {
+  const { options: { customServer, customServerUrl } } = getState();
+  const from = customServer ? customServerUrl : undefined;
```

Comment:
we might need a selector for it, I remember we have such code in code base a few times already.